client/src/__tests__/App.test.js

```diff
@@ -1,12 +1,19 @@
-import React from 'react'
-import {render, fireEvent, cleanup, waitForElement} from 'react-testing-library'
+import React from "react";
+import {
+  render,
+  fireEvent,
+  cleanup,
+  waitForElement
+} from "react-testing-library";
 
 // this adds custom jest matchers from jest-dom
-import 'jest-dom/extend-expect'
-import App from '../App';
+import "jest-dom/extend-expect";
+import App from "../App";
 
 afterEach(cleanup);
 
+it("CheckboxWithLabel changes the text after click", async () => {
+  const { getByText } = render(<App />);
```

Comment:
Incomplete test. It doesn't look like the function closes. Causes tests to fail

Reply:
this was done by someone else, Austin and I fixed it